### PR TITLE
Use KAS_MACHINE var

### PR DESCRIPTION
**Pull request by DaniilKl**
Related to https://github.com/zarhus/meta-zarhus/pull/44.

### CODE REVIEW

File: docs/getting-started/building.md [hunk 1]
```diff
@@ -52,26 +52,40 @@ This guide will demonstrate how to build a Zarhus OS image from zero!
...
 ```shell
-$ SHELL=/bin/bash kas-container build meta-zarhus/kas-IMAGE_TYPE.yml
+SHELL=/bin/bash KAS_MACHINE=<TARGET_NAME> kas-container build <KAS_FILES>
 ```
 
 !!! note
 
-    Replace `IMAGE_TYPE` with either `debug` or `prod`.
+    Replace `<TARGET_NAME>` with the name of the chosen target
+    configuration file, and `<KAS_FILES>` with a list of kas files, separated by
```
Comment (m-iwanicki) [inline docs/getting-started/building.md:70]: Do we have explanation about available `<KAS_FILES>` and which ones are needed for what and in which order (e.g. debug needs to be after common)?
Maybe we should include section for those and link there?
And depending on whether `debug.yml` is included target filename might change e.g. `-debug` in `zarhus-base-image-debug-...` might be or not be there.

File: mkdocs.yml [hunk 2]
```diff
@@ -24,12 +24,12 @@ extra:
...
-    # - icon: fontawesome/brands/twitter
-    #   link: https://twitter.com/TBD
-    # - icon: fontawesome/brands/reddit
-    #   link: https://www.reddit.com/r/TBD
+      # - icon: fontawesome/brands/mastodon
+      #   link: https://fosstodon.org/@TBD
+      # - icon: fontawesome/brands/twitter
+      #   link: https://twitter.com/TBD
+      # - icon: fontawesome/brands/reddit
+      #   link: https://www.reddit.com/r/TBD
```
Comment (m-iwanicki) [inline mkdocs.yml:32]: If you uncomment those lines (`# `) then they'll have wrong indent.
pre-commit doesn't complain unless there is also another error then it'll show warning so we could leave it as it was?

**Review by m-iwanicki — CHANGES_REQUESTED** [review]

File: mkdocs.yml (hunk 2 above)
Comment (DaniilKl) [reply mkdocs.yml:32]: > If you uncomment those lines (# ) then they'll have wrong indent

Then, when uncommenting, the indent should be adjusted, `yamllint` requires starting space in a comment. 

> pre-commit doesn't complain unless there is also another error then it'll show warning so we could leave it as it was?

I saw the warning - I fixed the warning. Why leaving it?

File: docs/getting-started/building.md (hunk 1 above)
Comment (DaniilKl) [reply docs/getting-started/building.md:81]: Added here c7220ee098cd81c3616cb6975047eb14c31285ba. Other inf. user could get in kas documentation

**Review by m-iwanicki — APPROVED** [review]

**Pull request merged**